build/lin.sh

```diff
@@ -322,15 +320,15 @@ mkdir ${DEPS}/webp
 $CURL https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-${VERSION_WEBP}.tar.gz | tar xzC ${DEPS}/webp --strip-components=1
 cd ${DEPS}/webp
 ./configure --host=${CHOST} --prefix=${TARGET} --enable-static --disable-shared --disable-dependency-tracking \
-  --disable-neon --enable-libwebpmux --enable-libwebpdemux
+  --enable-libwebpmux --enable-libwebpdemux ${WITHOUT_NEON:+--disable-neon}
 make install-strip
 
 mkdir ${DEPS}/tiff
 $CURL https://download.osgeo.org/libtiff/tiff-${VERSION_TIFF}.tar.gz | tar xzC ${DEPS}/tiff --strip-components=1
 cd ${DEPS}/tiff
-./configure --host=${CHOST} --prefix=${TARGET} --enable-static --disable-shared --disable-dependency-tracking \
-  --disable-mdi --disable-pixarlog --disable-old-jpeg --disable-cxx --disable-lzma --disable-zstd \
-  --with-jpeg-include-dir=${TARGET}/include --with-jpeg-lib-dir=${TARGET}/lib
+# Propagate -pthread into CFLAGS to ensure WebP support
+CFLAGS="${CFLAGS} -pthread" ./configure --host=${CHOST} --prefix=${TARGET} --enable-static --disable-shared --disable-dependency-tracking \
```

Comment:
This is a bit unfortunate. libtiff doesn't use pkg-config to find libwebp (so it's unaware of `Libs.private: -pthread`), which leads to a link failure when it checks for WebP support:
```
configure:19178: checking for WebPDecode in -lwebp
configure:19203: gcc -o conftest -march=westmere -Os -fPIC -D_GLIBCXX_USE_CXX11_ABI=1 -fno-asynchronous-unwind-tables -ffunction-sections -fdata-sections -Wall -W -DTIFF_DISABLE_DEPRECATED -I/target/include -L/target/lib -Wl,--gc-sections -Wl,-rpath='$$ORIGIN/' conftest.c -lwebp  -ljpeg -lz -lm  >&5
/opt/rh/devtoolset-11/root/usr/libexec/gcc/x86_64-redhat-linux/11/ld: /target/lib/libwebp.a(thread_utils.o): in function `End':
(.text.End+0x1c): undefined reference to `pthread_join'
/opt/rh/devtoolset-11/root/usr/libexec/gcc/x86_64-redhat-linux/11/ld: /target/lib/libwebp.a(thread_utils.o): in function `Reset':
(.text.Reset+0x82): undefined reference to `pthread_create'
collect2: error: ld returned 1 exit status
configure:19203: $? = 1
configure: failed program was:
```
(somehow the OSS-Fuzz build didn't require this)